MangoAPI.Client/src/app/components/chats/chats.component.ts

```diff
@@ -57,8 +59,12 @@ export class ChatsComponent implements OnInit {
   private signalRConnected = false;
   public realTimeConnections: string[] = [];
   public userId: string | undefined = '';
+  public userData: Tokens | undefined;
```

Comment:
This variable can be skipped because tokens can be got from `_tokensService` directly